File: Exemplos/App_eXperience_ReactNative/Elgin_AppExperience_ReactNative-font/android/app/src/main/java/com/elginm8/Pix4/Produto.java
package com.elginm8.Pix4;

public class Produto {
    public String nome;
    public String preco;
    public String assetFileName;
    public String outputFileName;

    public Produto(String nome, String preco, String assetFileName, String outputFileName) {
        this.nome = nome;
        this.preco = preco;
        this.assetFileName = assetFileName;
        this.outputFileName = outputFileName;
    }

    public static Produto[] values() {
        Produto productList[] = new Produto[]{
                new Produto("ABACAXI", "7.00", "pix4_image_abacaxi", "P1.jpg"),
                new Produto("BANANA", "7.00", "pix4_image_banana", "P2.jpg"),
                new Produto("CHOCOLATE", "7.00", "pix4_image_chocolate", "P3.jpg"),
                new Produto("DETERGENTE", "7.00", "pix4_image_detergente", "P4.jpg"),
                new Produto("ERVILHA", "7.00", "pix4_image_ervilha", "P5.jpg"),
                new Produto("FEIJAO", "7.00", "pix4_image_feijao", "P6.jpg"),
                new Produto("GOIABADA", "7.00", "pix4_image_goiabada", "P7.jpg"),
                new Produto("HAMBURGUER", "7.00", "pix4_image_hamburguer", "P8.jpg"),
                new Produto("IOGURTE", "7.00", "pix4_image_iogurte", "P9.jpg"),
                new Produto("JACA", "7.00", "pix4_image_jaca", "P10.jpg")
        };
        return productList;
    }
}

File: Exemplos/App_eXperience_Kotlin/Elgin_AppExperience_Kotlin-font/app/src/main/java/com/example/e1_kotlin_r/Services/Pix4/Produto.kt
package com.example.e1_kotlin_r.Services.Pix4

import com.example.e1_kotlin_r.R

enum class Produto(val preco: String, val fileName: String, val fileImageResId: Int) {
    ABACAXI("7.00", "P1.jpg", R.drawable.pix4_image_abacaxi),
    BANANA("8.50", "P2.jpg", R.drawable.pix4_image_banana),
    CHOCOLATE("10.00", "P3.jpg", R.drawable.pix4_image_chocolate),
    DETERGENTE("6.00", "P4.jpg", R.drawable.pix4_image_detergente),
    ERVILHA("4.00", "P5.jpg", R.drawable.pix4_image_ervilha),
    FEIJAO("8.00", "P6.jpg", R.drawable.pix4_image_feijao),
    GOIABADA("9.00", "P7.jpg", R.drawable.pix4_image_goiabada),
    HAMBURGUER("14.00", "P8.jpg", R.drawable.pix4_image_hamburguer),
    IOGURTE("11.00", "P9.jpg", R.drawable.pix4_image_iogurte),
    JACA("9.00", "P10.jpg", R.drawable.pix4_image_jaca);
}